src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,14 +1,29 @@
 import React from 'react';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { queryChange, statusChange } from '../../features/filter';
+
+type Props = {};
```

Comment:
The Props type is defined but not used. If no props are needed, you can remove this type definition.